### ISSUE TITLE
cannot relative-require typescript from javascript outside root project dir

### ISSUE THREAD

**Issue by amonks** [wallabyjs/quokka#169]
### Issue description or question

```
`require('../a')` from b fails here
.
└── dir
    ├── a.ts
    └── b.js
```

it succeeds if any of:
- a and b are typescript
- a and b are javascript
- b is in ., not dir

### Quokka.js Console Output

```
​​​​Quokka #1 (node: v8.1.2)​​​​
​​​​
​​Cannot find module './a'​​
  at ​​​Object.<anonymous>​​​ ​dir/b.js:1:0​
```

### Code editor version

Visual Studio Code Version 1.22.0-insider (1.22.0-insider)

### OS name and version

OSX 10.12.6

**Comment by ArtemGovorov** [comment]
With this file structure:

```
.
└── dir
    ├── a.ts
    └── b.js
```

`require('../a')` from `b` will fail regardless of the file extension, because `a` will be searched not on the same level where the `b` is, so you probably mean `require('./a')` or `require('../dir/a')`?

There was one scenario that wasn't supported. As you have pointed out, it wasn't possible to require `.ts` from `.js`. We have just released a new version 1.0.113 and now if you add the [`{ ts: true }` setting to any of 3 Quokka configuration sources](https://quokkajs.com/docs/configuration.html), then it should work for you. For example in your `b.js` you may do:

```
({
    ts: true
})
require('./a')
```

**Comment by amonks** [comment]
Thanks for the quick resolution!

I mistyped—I meant `require('./a')` (that's how I had it—you can see in the console output)

It actually _was_ possible to require `ts` from `js`, provided that you used paths relative to the the project root directory.

I'll try again on the new build with the ts: true setting and report back if anything still seems amiss.

***

_Huge_ fan of your projects btw. Great stuff!
